cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -213,7 +213,7 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
       text = "If it is unknown if a constraint exists or not but we want to make sure it does, we add the `IF NOT EXISTS`. " +
         "The node property existence constraint ensures that all nodes with a certain label have a certain property.",
       prepare = _ => executePreparationQueries(List("CREATE CONSTRAINT constraint_name IF NOT EXISTS FOR (book:Book) REQUIRE book.isbn IS UNIQUE")),
-      queryText = "CREATE CONSTRAINT constraint_name IF NOT EXISTS FOR (book:Book) REQUIRE book.isbn IS NOT NULL",
+      queryText = "CREATE CONSTRAINT constraint_name IF NOT EXISTS FOR (book:Book) REQUIRE book.isbn IS NOT NULL OPTIONS {}",
```

Comment:
Even if it is possible to add `OPTIONS {}` here for completeness, I would not include it in the example. Since it isn't required for doing IF NOT EXISTS but just redundant syntax to type, I think it is not helpful for users.

Reply:
I kind of wanted to have one example using it but not enough to add a whole new one but I agree with your point as well